polaris-react/src/components/ContentBlock/ContentBlock.scss

```diff
@@ -0,0 +1,12 @@
+.ContentBlock {
+  display: block;
```

Comment:
Is this line necessary?

Reply:
Without using `block` it seems I cant get the children to center correctly, and components like `Box` end up not using the full width of the container

Reply:
You're right! I was replacing it with `display: flex` instead of just removing it all together. Looks like its working correctly now 👍🏾 Thanks!